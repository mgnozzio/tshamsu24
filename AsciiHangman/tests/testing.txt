testing
T
e
s
s
s
t
i
n
G
