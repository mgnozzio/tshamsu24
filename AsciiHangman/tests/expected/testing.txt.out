Welcome to the ASCII Version of Hangman!
For best results, please run this from the command line.
Please enter a secret word: 
















































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































   ____  
  |    | 
  |      
  |      
  |      
__|__    
Puzzle to solve: _ _ _ _ _ _ _ 
Please guess a letter: T
Puzzle to solve: T _ _ T _ _ _ 
Please guess a letter: E
Puzzle to solve: T E _ T _ _ _ 
Please guess a letter: S
Puzzle to solve: T E S T _ _ _ 
Please guess a letter: S
Puzzle to solve: T E S T _ _ _ 
Please guess a letter: S
Puzzle to solve: T E S T _ _ _ 
Please guess a letter: T
Puzzle to solve: T E S T _ _ _ 
Please guess a letter: I
Puzzle to solve: T E S T I _ _ 
Please guess a letter: N
Puzzle to solve: T E S T I N _ 
Please guess a letter: G
Success!  Player 2 wins!
